### PR TITLE
Dev/karlo

### PR DESCRIPTION
**Pull request by MezdicKarlo**

### CODE REVIEW

File: frontend/src/routes/Login.jsx [hunk 1]
```diff
@@ -1,18 +1,74 @@
...
+                <input className='passwordInput' placeholder='Password' name='password' onChange={onChange} type='password' value={loginForm.password}/>
+            </div>
+            <div className='buttonDiv'>
+              <button className='inputButton' type='submit'>Sign In</button>
+              </div> 
+            <div className='redirect'>
+              <p className='account'>Dont have account?</p>
+              <a href='/users/register' className='link'>Register here</a>
+            </div>
+						<div className='oauth'>
```
Comment (savipauk) [inline frontend/src/routes/Login.jsx:60]: Weird indentation here, please fix

File: frontend/src/routes/Register.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+function Root() {
+
+  return (
+    <>
+      <RegisterForm></RegisterForm>
+    </>
+  )
+}
+
+function RegisterForm(){
```
Comment (savipauk) [inline frontend/src/routes/Register.jsx:13]: Whitespace before { -> `function RegisterForm() {`

File: frontend/src/routes/Register.jsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+          <h2 className='createNewText'>Create new account</h2>
+          <div className='inputDiv'>
+            <input className='infoInput' type='text' placeholder='Username' onChange={onChange} value={registerForm.username} name='username'/>
+            <input className='infoInput' type="text" placeholder='Email' onChange={onChange} value={registerForm.email} name='email'/>
+          </div>
+          <div className='passwordDiv'>
+            <input className='passwordInput' type='password' placeholder='Password' onChange={onChange} value={registerForm.password} name='password'/>
+            <input className='passwordInput' type='password' placeholder='Confirm password' onChange={onChange} value={registerForm.confirmPassword} name='confirmPassword'/>
+          </div>
+					<p className='errorMessage'>{errorMessage}</p>
```
Comment (savipauk) [inline frontend/src/routes/Register.jsx:72]: weird indentation

File: frontend/src/routes/Root.jsx [hunk 4]
```diff
@@ -1,32 +1,33 @@
 import { useState } from 'react'
-import reactLogo from '../assets/react.svg'
-import viteLogo from '/vite.svg'
 import '../styles/Root.css'
+import { useNavigate } from 'react-router-dom'
 
 function Root() {
   const [count, setCount] = useState(0)
```
Comment (savipauk) [inline frontend/src/routes/Root.jsx:6]: this can be removed

File: frontend/src/styles/Login.css [hunk 5]
```diff
@@ -0,0 +1,152 @@
+body{
+  /* background: linear-gradient(#c31431cb, #240b36e1); */
+	background: linear-gradient(#355C7D, #6C5B7B, #C06C84);
+
```
Comment (savipauk) [inline frontend/src/styles/Login.css:4]: remove blank line

File: frontend/src/styles/Root.css [hunk 6]
```diff
@@ -40,3 +35,61 @@
...
+	border: 0px;
+}
+
+#error-page{
+	display: flex;
+	justify-content: center;
+	width: 100vw;
+	flex-wrap: wrap;
+}
+.er{
```
Comment (savipauk) [inline frontend/src/styles/Root.css:92]: It's generally better to be more verbose. I would rename this to `errorText` as it's used for `<h1>` and `<p>` as far as I can tell

**Review by savipauk — CHANGES_REQUESTED** [review]
looks good but please fix formatting

File: frontend/src/routes/Root.jsx (hunk 4 above)
Comment (MezdicKarlo) [reply frontend/src/routes/Root.jsx]: error occurs if i remove this line

**Review by savipauk — APPROVED** [review]

**Pull request merged**